pt/lessons/basics/mix.md

```diff
@@ -1,23 +1,29 @@
 ---
-version: 1.1.1
+version: 1.1.2
 title: Mix
 ---
 
-Antes de podermos mergulhar nas águas mais profundas de Elixir primeiro precisamos aprender a utilizar o mix. Se você estiver familiarizado com Ruby, mix é o Bundler, RubyGems e o Rake juntos. É uma parte crucial de qualquer projeto Elixir e nesta lição vamos explorar apenas algumas das suas grandes funcionalidades. Para ver tudo que o mix tem para oferecer, execute `mix help`.
+Antes de podermos mergulhar nas águas mais profundas de Elixir primeiro precisamos aprender a utilizar o Mix. 
```

Comment:
```suggestion
Antes de podermos mergulhar nas águas mais profundas de Elixir, primeiro precisamos aprender a utilizar o Mix. 
```

Reply:
Thanks for the review, and I fixed the typo.